homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -213,34 +213,46 @@ def _async_register_bridge_device(
 def _async_register_button_devices(
     hass: HomeAssistant,
     config_entry_id: str,
+    bridge,
     bridge_device,
     button_devices_by_id: dict[int, dict],
-) -> dict[str, dict]:
+) -> tuple[dict[str, dict], dict[int, dict[str, Any]]]:
     """Register button devices (Pico Remotes) in the device registry."""
     device_registry = dr.async_get(hass)
     button_devices_by_dr_id: dict[str, dict] = {}
+    device_info_by_device_id: dict[int, dict[str, Any]] = {}
     seen = set()
```

Comment:
```suggestion
    seen: set[str] = set()
```

Are these always strings?

Reply:
oh, I see.  That was existing.  Yes, it will always be a string.  Serial number of the Keypad.

Reply:
no problem.